addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AutomationPanel.java

```diff
@@ -167,7 +167,7 @@ public class AutomationPanel extends AbstractPanel implements EventConsumer {
     public AutomationPanel(ExtensionAutomation ext) {
         this.ext = ext;
         this.setName(Constant.messages.getString("automation.panel.title"));
-        this.setIcon(ExtensionAutomation.ICON);
+        this.setIcon(ExtensionAutomation.getRobotIcon());
```

Comment:
This doesn't cause a further exception when trying to set null?

Reply:
This piece of code will only get called if the desktop is being displayed, so it should be fine 😁

Reply:
Since it's used only here, why not create it here instead?

Reply:
Too obvious 😛 
Updated..